### PR TITLE
Ols 212: Missing documentation about index store

### PR DESCRIPTION
**Pull request by sauagarwa**
## Description
When asking "what is kubernetes?" with the OpenAI backend, the server crashes with FileNotFoundError: [Errno 2] No such file or directory: '/home/ometelka/projects/lightspeed-service/vector-db/ocp-product-docs/docstore.json' 

## Type of change

- [ ] Refactor
- [ ] New feature
- [ X] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue # [OLS-212](https://issues.redhat.com//browse/OLS-212)
- Closes # [OLS-212](https://issues.redhat.com//browse/OLS-212)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by onmete** [comment]
For context, I was encountering the exception when using
```
DEFAULT_PROVIDER=openai
DEFAULT_MODEL=gpt-3.5-turbo
```
I can confirm that I can prompt now.

I get an empty JSON back as a response. I guess it makes sense as it searches empty index store, but can @bparees confirm its intended behavior?

File: ols/src/docs/docs_summarizer.py [hunk 1]
```diff
@@ -72,16 +73,21 @@ def summarize(self, conversation: str, query: str, **kwargs) -> tuple[str, str]:
...
+            index = load_index_from_storage(
+                storage_context=storage_context,
+                index_id=constants.PRODUCT_INDEX,
+                service_context=service_context,
+                verbose=verbose,
+            )
+        except FileNotFoundError as err:
+            self.logger.error(f"FileNotFoundError: {err.strerror}, file= {err.filename}")
+            self.logger.info("Initializing empty vector index.")
+            index = VectorStoreIndex([])
```
Comment (bparees) [inline ols/src/docs/docs_summarizer.py:90]: discussed on scrum, if we have no vector index we should not setup a query engine, and we also need to send back additional information in the response object indicating that the response was generated w/o access to RAG content.

**Comment by bparees** [comment]
@sauagarwa you need to join the openshift github org so your PRs can be automatically tested.  Submit a ticket here:
https://devservices.dpp.openshift.com/support/github_access_request/

/ok-to-test

**Pull request merged**